task2/app.py

```diff
@@ -3,32 +3,52 @@
 # 
 
 class PhoneBook:
+    
     def __init__(self) -> None:
-        pass
+        self.contacts = []
 
     def add(self, name: str, surname: str, phone: str) -> None:
-        """??? (https://peps.python.org/pep-0257/)"""
-        pass
+        """Adds a person to the phone book."""
+        for contact in self.contacts:
+            if contact['name'] == name and contact['surname'] == surname:
+                return
+        self.contacts.append({'name': name, 'surname': surname, 'phone': phone})
 
     def update(self, name: str, surname: str, phone: str) -> bool:
-        """??? (https://peps.python.org/pep-0257/)"""
-        pass
+        """Updates information about a person in the phone book."""
+        for contact in self.contacts:
+            if contact['name'] == name and contact['surname'] == surname:
+                contact['phone'] = phone
```

Comment:
Note that  this function is expected to return `bool` (look at the line 18), but you return nothing. Please, fix it

Reply:
done